utils/gen-corim/cmd/gen-corim.go

```diff
@@ -15,34 +15,51 @@ import (
 
 	"github.com/lestrrat-go/jwx/v2/jwk"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 	"github.com/veraison/ccatoken"
 	"github.com/veraison/corim/comid"
 	"github.com/veraison/eat"
 	"github.com/veraison/psatoken"
 )
 
 var (
-	cfgFile                  string
-	coevcliAttestationScheme *string
-	coevcliEvidenceFile      *string
-	coevcliKeyFile           *string
-	coevcliCorimFile         *string
-	coevcliTemplateDir       *string
+	genCorimAttestationScheme *string
+	genCorimEvidenceFile      *string
+	genCorimKeyFile           *string
+	genCorimCorimFile         *string
+	genCorimTemplateDir       *string
 )
 
 var rootCmd = NewRootCmd()
 
 func NewRootCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "coevcli",
-		Short:   "create corim from supplied evidence",
+		Use:   "gen-corim",
+		Short: "generate CoRIM from supplied evidence",
+		Long: `generate CoRIM from supplied evidence
+		
+		Generate CoRIM from evidence token (evidence.cbor), attestation scheme to use (only schemes supported 
+		by ths tool are psa and cca) and key material needed to verify the evidence (key.json). 
+		Save it to the current working directory with default file name.
+
```

Comment:
@Kakemone Are these the only arguments.
I think, we need the supplied directory as an argument also, isn't it?